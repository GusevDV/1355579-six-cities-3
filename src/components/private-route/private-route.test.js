import React from "react";
import renderer from "react-test-renderer";
import {BrowserRouter} from 'react-router-dom';
import PrivateRoute from "./private-route.jsx";
import {Provider} from "react-redux";
import configureStore from "redux-mock-store";

const mockStore = configureStore([]);
const store = mockStore({
  user: {
    isAuthrorized: true,
    data: []
  }
});

it(`Should Footer component render correctly`, () => {

  const render = renderer.create(
      <Provider store={store}>
        <BrowserRouter>
          <PrivateRoute exact={true} path={`path`} render={() => (
            <div></div>
          )} />
        </BrowserRouter>
      </Provider>
  )
  .toJSON();

  expect(render).toMatchSnapshot();
});

