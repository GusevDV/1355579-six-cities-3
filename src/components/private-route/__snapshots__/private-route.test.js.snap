// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Footer component render correctly 1`] = `null`;
